logger/logger.go

```diff
@@ -0,0 +1,47 @@
+package logger
+
+import (
+	"io"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+// NewLogger creates a new logger that can write to disk.
+//
+// The function also returns a closer function that should be called when we
+// stop using the logger, typically deferred in main.
+func NewLogger(level, logfile string) (logger *logrus.Logger, closer func(), err error) {
+	logger = logrus.New()
+	// Parse and set log level.
+	logLevel, err := logrus.ParseLevel(level)
```

Comment:
This still feels like it would belong into `LoadConfig` tbh.  There is not reason to not make the LogLevel field of the Config a `logrus.LogLevel` and immediately parse the environment var.
The sooner we turn vague user input into strong types the better.